labop_convert/markdown/markdown_specialization.py

```diff
@@ -1006,6 +1008,9 @@ def measurement_to_text(measure: sbol3.Measure):
     return f'{measurement_scalar} {measurement_units}'
 
 
+def dataset_to_text(dataset: labop.Dataset):
+    return f'{dataset.display_name}'
```

Comment:
I'm guessing there should be a "TODO" here for expanding this in some way?

Reply:
Yes, I had this in my head, but didn't TODO it.